core/arch/arm/plat-sam/main.c

```diff
@@ -142,11 +572,12 @@ static void matrix_configure_slave_h64mx(void)
 			| MATRIX_WRNSECH_NS(3));
 	/* DDR port 0 not used from NWd */
 	for (ddr_port = 1; ddr_port < 8; ddr_port++) {
-		matrix_configure_slave_security(matrix64_base(),
-					(H64MX_SLAVE_DDR2_PORT_0 + ddr_port),
-					srtop_setting,
-					sasplit_setting,
-					ssr_setting);
+		matrix_configure_slave_security(matrix_base(MATRIX_H64MX),
+						(H64MX_SLAVE_DDR2_PORT_0
+						+ ddr_port),
```

Comment:
Prefer operator at the end of the previous line.
Can remove the inner parentheses.

Could prevent a line break:
```c
		matrix_configure_slave_security(matrix_base(MATRIX_H64MX),
						H64MX_SLAVE_DDR2_PORT_0 +
						ddr_port, srtop_setting,
						sasplit_setting, ssr_setting);
```

Reply:
Fixed, thanks.